conanfile.py

```diff
@@ -58,11 +69,10 @@ class Xrpl(ConanFile):
     }
 
     requires = [
+        'doctest/2.4.6',
```

Comment:
I don't think we need this.

Reply:
I don't remember why this snuck in :thinking: